libbeat/metric/system/memory/memory.go

```diff
@@ -69,6 +69,14 @@ func GetSwap() (*SwapStat, error) {
 		return nil, err
 	}
 
+	// This shouldn't happen, but it has been reported to happen and
+	// this can provoke too big values for used swap.
+	// Workaround this by assuming that all swap is free in that case.
+	if swap.Free > swap.Total || swap.Used > swap.Total {
```

Comment:
I would suggest to at leats log something on the debug level in this cases. So if we want to investigate this further in the future, we can see the actual values reported.

Reply:
Agree